lib/csa/ccm/cli/command.rb

```diff
@@ -11,24 +11,24 @@ class Command < Thor
         desc 'ccm-yaml VERSION', 'Generating a machine-readable CCM/CAIQ'
         option :output_file, aliases: :o, type: :string, desc: 'Optional output YAML file. If missed, the input file’s name will be used'
 
-        def ccm_yaml(version)
-          input_files = Resource.lookup_version(version)
+        def ccm_yaml(caiq_version)
+          input_files = Resource.lookup_version(caiq_version)
 
-          unless input_files || !input_files.empty?
-            UI.say("No file found for #{version} version")
+          unless input_files && !input_files.empty?
+            UI.say("No file found for #{caiq_version} version")
             return
           end
 
           input_file = input_files.first
 
           unless File.exist? input_file
-            UI.say("#{input_file} doesn't exists for #{version} version")
+            UI.say("#{input_file} doesn't exists for #{caiq_version} version")
             return
           end
 
-          matrix = Matrix.from_xlsx(version, input_file)
+          matrix = Matrix.from_xlsx(caiq_version, input_file)
```

Comment:
@CAMOBAP795 I don't think you even need to pass `caiq_verssion` anymore because it can already run `Matrix.parse_version`.